<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjout</class>
 <widget class="QDialog" name="DialogAjout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout d'un document</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choisir le type de document</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manga</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Comic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Roman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Article</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="leTitle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>titre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAuthor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>auteur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTheme">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>theme</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDateParution">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>date parution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEditor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>editeur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leType">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbOk">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
